source/calculus/source/09-PS/05.ptx

```diff
@@ -0,0 +1,361 @@
+                
+            </p>
+        </introduction>
+        <task>
+            <statement>
+                <p>
+                    Compute <m>R_2(4)</m> using technology.
+                </p>
+            </statement>
+        </task>
+        <task>
+            <statement>
+                <p>
+                    Compute <m>R_3(4)</m> using technology.
+                </p>
+            </statement>
+        </task>
+        <task>
+            <statement>
+                <p>
+                    What do you expect from <m>R_4(4)</m>?
+                </p>
+                <ol marker="A.">
+                    <li>
+                        <p>
+                            There is not enough information.
+                        </p>
+                    </li>
+                    <li>
+                        <p>
+                            It will be greater than both <m>R_3(4)</m> and <m>R_4(4)</m>.
+                        </p>
+                    </li>
+                    <li>
+                        <p>
+                            It will be between <m>R_3(4)</m> and <m>R_4(4)</m>.
+                        </p>
+                    </li>
+                    <li>
+                        <p>
+                            It will be less than both <m>R_3(4)</m> and <m>R_4(4)</m>.
+                        </p>
+                    </li>
+                </ol>
+            </statement>
+        </task>
+    </activity>
+
+   <fact xml:id="fact-remainder-goes-to-zero">
+    <statement>
+        <p>
+            Let <m>f(x)</m> be a function represented by a power series centered at <m>x=c</m>
+            <me>
+                f(x)=\displaystyle\sum_{n=0}^\infty a_n(x-c)^n
+            </me>
+            with an interval of convergence <m>I</m>. Then for all <m>x</m> in <m>I</m>,
+            <me>
+                \lim_{k\rightarrow\infty} R_k(x)=0.
+            </me>
+            
+        </p>
+    </statement>
+   </fact> 
+
+    <theorem xml:id="thm-taylors-theorem">
+        <title>Taylor's Theorem</title> 
+        
+        <statement>
+            <p>
+                Let <m>f(x)</m> be an <m>(k+1)</m> times differentiable function on an interval <m>I</m> of <m>a</m>, and let <m>T_k(x)</m>
+                be its <m>k</m>th degree Taylor polynomial centered at <m>x=a</m>. Then for any <m>x</m> in the interval <m>I</m>, 
+                there exists <m>c</m> between <m>a</m> and <m>x</m> such that
+                <me>
+                    R_k(x)=\dfrac{f^{(k+1)}(c)}{(k+1)!}(x-a)^{k+1}.
+                </me>
+                If there exists <m>M_k</m> such that <m>|f^{(k+1)}(x)|\leq M_k</m> for all <m>x</m> in <m>I</m>, then the error in 
+                the approximation <m>f(x)\approx T_k(x)</m> has an upper bound:
+                <me>
+                    |R_k(x)|\leq \dfrac{M_k}{(k+1)!}|x-a|^{k+1}.
+                </me>               
+                
+            </p>
+        </statement>
+    </theorem>
```

Comment:
Should the centering term be $x-c$ to be consistent with what's written in Fact 9.5.4?